tests/spread/integration/apache2-bin/task.yaml

```diff
@@ -0,0 +1,48 @@
+summary: Integration tests for apache2-bin
+
+execute: |
+  rootfs="$(install-slices apache2-bin_bins base-passwd_data)"
+
+  chroot "${rootfs}/" apache2 -v
+  mkdir -p "${rootfs}/dev"
+  mount --rbind /dev "${rootfs}/dev"
+  mkdir -p "${rootfs}/etc/apache2"
+  mkdir -p "${rootfs}/etc/apache2/logs"
+  mkdir -p "${rootfs}/var/run/apache2"
+  mkdir -p "${rootfs}/var/lock/apache2"
+  mkdir -p "${rootfs}/var/log/apache2"
+  mkdir -p "${rootfs}/var/www/html"
+
+  chown www-data:www-data "${rootfs}/var/run/apache2"
+  chown www-data:www-data "${rootfs}/var/lock/apache2"
+  chown www-data:www-data "${rootfs}/var/log/apache2"
+  chown -R www-data:www-data "${rootfs}/var/www"
+
+  chmod 755 "${rootfs}/var/run/apache2"
+  chmod 755 "${rootfs}/var/lock/apache2"
+  chmod 755 "${rootfs}/var/log/apache2"
+  chmod -R 755 "${rootfs}/var/www"
+
+  cat <<'EOF' > "${rootfs}/etc/apache2/apache2.conf"
+  ServerRoot "/etc/apache2"
+  ServerName localhost
+  Listen 80
+  LoadModule mpm_event_module /usr/lib/apache2/modules/mod_mpm_event.so
+  LoadModule authz_core_module /usr/lib/apache2/modules/mod_authz_core.so
+  LoadModule dir_module /usr/lib/apache2/modules/mod_dir.so
+  ErrorLog /etc/apache2/logs/error.log
+  User www-data
+  Group www-data
+  DocumentRoot "/var/www/html"
+  <Directory "/var/www/html">
+      Options Indexes FollowSymLinks
+      AllowOverride None
+      Require all granted
+  </Directory>
+  EOF
+
+  echo "Hello, Apache!" > "${rootfs}/var/www/html/index.html"
+  test "$(chroot "${rootfs}" apache2 -t 2>&1 )" = "Syntax OK"
+  chroot "${rootfs}" apache2 -k start
+  test "$(curl -s -o /dev/null -w "%{http_code}\n" http://127.0.0.1)" = "200"
```

Comment:
Looks like this step in the spread test fails in CI, can you review?
See https://github.com/norrisjeremy/chisel-releases/actions/runs/13926165273/job/38971117957#step:9:183.